### PR TITLE
chore: forward v1.24.2 release commit to main

### PR DESCRIPTION
**Pull request by denobot**
This is the release commit being forwarded back to main for 1.24.2

Please ensure:
- [x] Everything looks ok in the PR
- [x] The release has been published

To make edits to this PR:
```shell
git fetch upstream forward_v1.24.2 && git checkout -b forward_v1.24.2 upstream/forward_v1.24.2
```

Don't need this PR? Close it.

cc @crowlKats

### CODE REVIEW

File: cli/tests/unit/webcrypto_test.ts [hunk 1]
```diff
@@ -716,28 +716,27 @@ Deno.test(async function testAesCtrEncryptDecrypt() {
 });
 
 Deno.test(async function testECDH() {
```
Comment (dsherret) [inline cli/tests/unit/webcrypto_test.ts:718]: I think just revert the changes to this file and this PR should be good.

File: ext/crypto/lib.rs [hunk 2]
```diff
@@ -556,45 +556,9 @@ pub async fn op_crypto_derive_bits(
           // raw serialized x-coordinate of the computed point
           Ok(shared_secret.raw_secret_bytes().to_vec().into())
         }
-        CryptoNamedCurve::P384 => {
```
Comment (dsherret) [inline ext/crypto/lib.rs:559]: Oh, and here.

**Review by crowlKats — APPROVED** [review]
